### ISSUE TITLE
Consistent molecular weight of iron

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#88]
The molecular weight of iron is 55.845 u ± 0.002 u (source: [Commission on Isotopic Abundances and Atomic Weights](http://www.ciaaw.org/atomic-weights.htm); as of this posting, 2015 was the most recent report). This is the value used in `marbl_constants_mod.F90`

```
      molw_Fe   =  55.845_r8,          & ! molecular weight of iron
```

but in `marbl_parms.F90` we set

```
       dust_to_Fe=0.035_r8/55.847_r8*1.0e9_r8
```

where the `55.847_r8` term represents the molecular weight of iron*. We should use the `molw_Fe` variable in computing `dust_to_Fe`, but this will be answer-changing so we need determine a good time to make this fix.

\* Totally orthogonal to the discussion about what to do in MARBL, but I think it's interesting that from 1961 to 1992, the accepted molecular weight was 55.847 u ± 0.003 u. It seems the `dust_to_Fe` computation in MARBL can trace it's lineage to code either developed pre-1993 or at least to a time before the 1993 change to the standard was widely known. (Or perhaps just a developer who didn't realize they were looking at an outdated table.)

**Comment by mnlevy1981** [comment]
We will also need to update the comments: `marbl_mod` does not need to rehash the definition of the `dust_to_fe` value and `marbl_parms` should not use the verb "compute" in describing `dust_to_fe`.

**Comment by mnlevy1981** [comment]
closed via #183
